src/components/CandidateBox.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Candidate from './Candidate';
+import axios from 'axios';
+import { url } from './url';
+
+const CandidateBox = () => {
+  const [voteCandidate, setVoteCandidate] = useState('');
+  const getCandidates = async () => {
+    try {
+      const Candidates = await axios.get(`${url}/candidates`);
+      setVoteCandidate(Candidates.data);
+    } catch (e) {
+      console.error(e);
+    }
+  };
+  useEffect(() => {
+    getCandidates();
+  }, []);
+
+  return (
+    <Wrapper>
+      {voteCandidate &&
+        voteCandidate
+          .sort((a, b) => {
+            return b.voteCount - a.voteCount;
+          })
+          .map((c, index) => {
```

Comment:
약어를 쓰기보단 `candidate`라고 써주시는게 더 명확할 것 같습니당!

```suggestion
          .map((candidate, index) => {
```

Reply:
수정했습니다~!